sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -1841,8 +1873,16 @@
   @Deprecated
   public static final transient Metric<Integer> PACKAGE_FEEDBACK_EDGES = DIRECTORY_FEEDBACK_EDGES;
 
+  /**
+   * @deprecated since 5.2 No more design features
+   */
+  @Deprecated
   public static final String DIRECTORY_EDGES_WEIGHT_KEY = "package_edges_weight";
-  public static final Metric<Integer> DIRECTORY_EDGES_WEIGHT = new Metric.Builder(DIRECTORY_EDGES_WEIGHT_KEY, "Directory edges weight", Metric.ValueType.INT)
+  /**
+   * @deprecated since 5.2 No more design features
+   */
+  @Deprecated
+  public static final transient Metric<Integer> DIRECTORY_EDGES_WEIGHT = new Metric.Builder(DIRECTORY_EDGES_WEIGHT_KEY, "Directory edges weight", Metric.ValueType.INT)
```

Comment:
:new: ![INFO](https://raw.githubusercontent.com/henryju/image-hosting/master/info.png)Do not forget to remove this deprecated code someday.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)
:new: ![MINOR](https://raw.githubusercontent.com/henryju/image-hosting/master/minor.png)Remove the "transient" modifier from this field.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2065)